includes/enqueue-scripts.php

```diff
@@ -82,19 +82,15 @@ function edac_enqueue_scripts( $mode = '' ) {
 	);
 
 	if ( '' === $mode ) {
-		if ( ( current_user_can( 'edit_post', $post_id ) && ! is_customize_preview() )
-		) {
-			$mode = 'ui';
-		} else {
+		if ( ! current_user_can( 'edit_post', $post_id ) || is_customize_preview() ) {
 			return;
 		}
+		$mode = 'ui';
 	}
 
 	if ( ( 'full-scan' === $mode && $pro )
-			||
-			( 'ui' === $mode || 'editor-scan' === $mode
-			&&
-			$post_id && current_user_can( 'edit_post', $post_id ) )
+		|| 'ui' === $mode 
+		|| ( 'editor-scan' === $mode && $post_id && current_user_can( 'edit_post', $post_id ) )
```

Comment:
Refactored this condition a bit, because with the prioritization of `||`/`&&` it was not apparent at first glance what it does.